add-on/panels/toc.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/.  -->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <base href="chrome://loop/content">
+    <link rel="stylesheet" type="text/css" href="shared/css/reset.css">
+    <link rel="stylesheet" type="text/css" href="shared/css/common.css">
+    <link rel="stylesheet" type="text/css" href="shared/css/toc.css">
+  </head>
+  <body>
+
+    <div id="main"></div>
+
+    <script type="text/javascript" src="panels/vendor/l10n.js"></script>
+    <script type="text/javascript" src="shared/vendor/react.js"></script>
+    <script type="text/javascript" src="shared/vendor/lodash.js"></script>
+    <script type="text/javascript" src="shared/vendor/sdk.js"></script>
+    <script type="text/javascript" src="shared/vendor/backbone.js"></script>
+    <script type="text/javascript" src="shared/vendor/classnames.js"></script>
+
+
+    <script type="text/javascript" src="shared/js/loopapi-client.js"></script>
+    <script type="text/javascript" src="shared/js/utils.js"></script>
+    <script type="text/javascript" src="shared/js/urlRegExps.js"></script>
+    <script type="text/javascript" src="shared/js/mixins.js"></script>
+    <script type="text/javascript" src="shared/js/validate.js"></script>
+    <script type="text/javascript" src="shared/js/actions.js"></script>
+    <script type="text/javascript" src="shared/js/dispatcher.js"></script>
+    <script type="text/javascript" src="shared/js/otSdkDriver.js"></script>
+
+    <script type="text/javascript" src="shared/js/standaloneMozLoop.js"></script>
+
+    <!-- Stores need to be loaded before the views that uses them -->
+    <script type="text/javascript" src="shared/js/store.js"></script>
+    <script type="text/javascript" src="panels/js/roomStore.js"></script>
+    <script type="text/javascript" src="shared/js/activeRoomStore.js"></script>
+    <script type="text/javascript" src="shared/js/textChatStore.js"></script>
+    <script type="text/javascript" src="shared/js/standaloneAppStore.js"></script>
```

Comment:
standaloneAppStore & standaloneMozLoop don't seem to have moved according to what's in this patch, so I'm guessing these both are 404s at the moment.